providers/default.rb

```diff
@@ -24,8 +24,6 @@
 
 use_inline_resources
 
-include Chef::Mixin::ShellOut
```

Comment:
this is necessary for old chef-client versions before it was included in providers and resources

Reply:
oic you removed `shell_out!`